middle_end/flambda/inlining/inlining_decision.ml

```diff
@@ -125,7 +130,11 @@ module Function_declaration_decision = struct
   let report fmt t =
     Format.fprintf fmt "@[<v>The function %s be inlined at its use-sites@ \
                         because @[<hov>%a@]@]"
-      (if can_inline t then "can" else "cannot") report_reason t
+      (match to_result t with
+       | Always_inlinable -> "should always"
+       | Maybe_inlinable -> "maybe"
```

Comment:
```suggestion
       | Maybe_inlinable -> "may"
```

considering the rest of the message (i.e. the current code would print `The function maybe be inlined ....`)